craft_parts/packages/apt_cache.py

```diff
@@ -235,7 +235,7 @@ def get_installed_version(
                 return installed.version
         return None
 
-    def fetch_archives(self, download_path: Path) -> List[Tuple[str, str, Path]]:
+    def fetch_archives(self, download_path: Path) -> list[tuple[str, str, Path]]:
```

Comment:
I was going to comment on the `List` vs `list`, but apparently this file has had the `__future__.annotations` import for over two years now! Weird

Reply:
I removed `__future__.annotations` for now to keep the style the same with other files.